requirements.txt

```diff
@@ -1,3 +1,4 @@
 # the default package dependencies
 -r ./requirements/app/base.txt
 -r ./requirements/pytorch/base.txt
+-r ./requirements/lite/base.txt
```

Comment:
Is this necessary? Do we want the general installation to install Lite by default?

PL would include Lite's dependencies

Reply:
I'm just trying to put together a PR, hoping someone will help me get this stuff right.

I already cried for help on Slack because I'm uncertain about these things 😅 . I wish someone else could take these decisions off of me.

Reply:
happy to help :)

Reply:
Okay thanks for the help. So should I remove the

`-r ./requirements/lite/base.txt`

from this file then?

Reply:
I think so :)